src/components/menu/menu.d.ts

```diff
@@ -1,9 +1,22 @@
-import * as React from 'react';
+import * as React from "react";
+
+interface MenuContextProps {
+  menuType: "light" | "dark";
```

Comment:
These should all be optional I think

Reply:
these are the context prop, as far as I have looked they have no checks if a prop exists everywhere they are used